### DISCUSSION TITLE
Median and CI results are incorrect between lifelines and Proc lifetest procedure from SAS

### DISCUSSION THREAD

**Discussion by indra3007** [CamDavidsonPilon/lifelines#1548] [Q&A]
Hi, I am comparing the results between the sas procedure PROC LIFETEST and LIFELINES for KM estimates. The results are incorrect. How can I get the correct results?

**Comment by CamDavidsonPilon** [comment]
You'll have to provide more information: dataset, results, code, etc.

**Reply by indra3007** [reply]
Here are the results. 

Code and Results from python. 
df_subset = df[df["TRT01PN"] == 81]
df.to_excel("/Users/invarma/Desktop/streamlit/test.xlsx", index=False)
T = df_subset["AVAL"]
E = df_subset["CNSR"]
    
    # Fit the Kaplan-Meier survival model
kmf = KaplanMeierFitter()
kmf.fit(durations=T, event_observed=E)
    
    # Calculate the median survival time
median = kmf.median_survival_time_
print(median)
Result = 167.0
    # Calculate the median survival time confidence interval
median_confidence_interval = median_survival_times(kmf.confidence_interval_)
print(median_confidence_interval)
Result =      
KM_estimate_lower_0.95  KM_estimate_upper_0.95
0.5                    63.0                   188.0

Sas code and results.

data all;
	set adtte;
	where trt01pn eq 81;
	keep subjid trt01pn aval cnsr;
run;
proc lifetest data=all  reduceout OUTSURV=ple conftype=loglog stderr;
time aval*cnsr(1);
strata trt01pn;
ods output Quartiles=surve; 
run;
Result : 
I have attached the screenshot.

Attached the data as well. 
[test.xlsx](https://github.com/CamDavidsonPilon/lifelines/files/12135183/test.xlsx)
<img width="1123" alt="Screenshot 2023-07-21 at 7 51 39 PM" src="https://github.com/CamDavidsonPilon/lifelines/assets/129800591/41f16dcd-42f6-465f-8194-894d9099a9f1">
